Sources/BraintreeSEPADirectDebit/SEPADebitAccountsRequest.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// The POST body for `v1/payment_methods/sepa_debit_accounts`
+struct SEPADebitAccountsRequest: Encodable {
```

Comment:
❓ - do we want to create a `/Models` directory within each `Sources/<PaymentMethod>` directory? Specifically to house these new classes specific to encoding (and maybe eventually to house ones specific to decoding if we go that route).

Reply:
Yeah, I think that would be helpful. I was thinking that as I was adding this. Should we do that as a follow up PR for all of the models we've created so far? I think it changes the source paths for the package managers (at least Cocoapods). Also down to do that for SEPA in this PR and just a follow up for the other 2. Thoughts?

Reply:
I like the idea of having those be 1 PR! But your call

Reply:
Cool - agree we can 🏈 on it in this PR to establish a pattern in a single PR moving forward